lib/converters/converter-v2-to-v1.js

```diff
@@ -304,15 +304,12 @@ _.extend(Builders.prototype, {
      */
     responses: function (item) {
         var self = this,
-            responses = [],
             allResponses = item.responses || item.response;
-        if (!allResponses) {
-            return;
-        }
-        _.each(allResponses, function (response) {
-            responses.push(self.response(response, item));
+
```

Comment:
@kunagpal ... sure this change works?

Reply:
Yes, this was causing a lint error, as an array was being pushed to inside a `_.each`, which should be replaced by `_.map`.